components/track/TrackContributors.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import type { TrackModel } from "@bcc-code/bmm-sdk-fetch";
+
+const showMediaPlayer = inject<boolean>("showMediaPlayer");
+
+defineProps<{
+  track: TrackModel;
+}>();
+</script>
+
+<template>
+  <div class="bg-background-2 rounded-lg">
+    <div v-for="(contributor, index) in track?.contributors" :key="index">
+      <NuxtLink
+        class="rounded-lg text-label-1 flex flow-row gap-3 p-2 px-5 hover:bg-background-3"
+        :to="`/playlist/contributor/${contributor.id}`"
```

Comment:
Please use an object-structure for navigation instead. This will avoid dead-links as we by type-safety can ensure those links actually exist in the router.

Reply:
done